.env.sample.mainnet

```diff
@@ -133,3 +133,6 @@ VE_LOCATOR_ADDRESS=0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
 
 # validator-ejector staking module ID
 VE_STAKING_MODULE_ID=1
+
+# Lido operator ID
+#VE_OPERATOR_ID=
```

Comment:
will this stay as a comment

Reply:
Yes, the guide will direct users to uncomment this through a CLI command.